aries-site/src/data/wcag/tag.json

```diff
@@ -0,0 +1,512 @@
+    "rule": "2.2.1",
+    "label": "Timing Adjustable",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.2",
+    "label": "Pause, Stop, Hide",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.3",
+    "label": "No Timing",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.4",
+    "label": "Interruptions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.5",
+    "label": "Re-authenticating",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.2.6",
+    "label": "Timeouts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.1",
+    "label": "Three Flashes or Below Threshold",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.2",
+    "label": "Three Flashes",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.3.3",
+    "label": "Animation from Interactions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.1",
+    "label": "Bypass Blocks",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.2",
+    "label": "Page Titled",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.3",
+    "label": "Focus Order",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.4",
+    "label": "Link Purpose (In Context)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.5",
+    "label": "Multiple Ways",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.6",
+    "label": "Headings and Labels",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.7",
+    "label": "Focus Visible",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.8",
+    "label": "Location",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.9",
+    "label": "Link Purpose (Link Only)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.10",
+    "label": "Section Headings",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.2",
+    "label": "Pointer Cancellation",
+    "status": "conditional",
+    "notes": "Marking conditional because it depends on the implementation, if there is an easy way to undo clicking a tag or closing it"
+  },
+  {
+    "rule": "2.5.3",
+    "label": "Label in Name",
+    "status": "passed",
```

Comment:
would this be passed? If we have a removable tag does the screen reader announce "remove tag (name)" I did not think but we can re check

Reply:
This rule is less about the screenreader and more about the accessible name that accessibility tech uses to refer to the component by. The accessible name should match what we see on the screen. See in the below screenshot the text we see on the screen is 'name : value' and the accessible name is 'name: value'. This helps users who use speech based tech to reference the correct component
<img width="956" alt="Screenshot 2025-03-31 at 12 07 55 PM" src="https://github.com/user-attachments/assets/6559417c-aa49-462c-bb9e-4c7ce75e7499" />

Reply:
good to know thank you